src/ReportManager/SchemaFormContext/SchemaForm/fields/Section/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { SchemaFormContext } from '../../SchemaFormContext';
+import Header from '../Header';
+import Text from '../Text';
+import { FORM_FIELDS_TYPES } from '../../../../constants';
+import useFieldDetails from '../../useFieldDetails';
+
+const SECTION_ITEM_TYPES = {
+  HEADER: 'header',
+  FIELD: 'field'
+};
+
+const FORM_FIELDS = {
+  [FORM_FIELDS_TYPES.TEXT]: Text
+};
+
+const Section = ({ sectionName }) => {
+  const { getFieldUIType, isFieldActive } = useContext(SchemaFormContext);
```

Comment:
Why are we receiving these method through context? Both of them sound like utilities. Sending them through context has the potential to cause rerenders. Static util methods don't.

Reply:
I have extract all those methods to a utils file, I'll let you know when the changes are pushed